process/request_resources.md

```diff
@@ -0,0 +1,60 @@
+# Requesting Resources
+
+All OpenSSF projects receive support and resources from the Foundation. The type of resources and amount vary by project stage; refer to the [project lifecycles](process/project_lifecycle.md) for details. 
+
+- Common requests
+    - [Blogs](#communications-and-content)
+    - [Event space](#contribuor-meet-ups-at-OpenSSF-events)
+    - [Infrastructure](#infrastructure)
+    - [Security audits](#security-audits-and-improvements)
+    - [Code of Conduct support](#code-of-conduct-support)
+
+## Requests through the OpenSSF Marketing Committee
+
+### Communications and content
+
+To post on the OpenSSF blog, projects should contact the OpenSSF Marketing Committee with a blog draft two weeks before the requested publication date. The Committee does their best to accommodate requested publication dates, but projects should plan to be flexible during high-traffic times (e.g. events). Posts and timing is at the editorial discretion of the Marketing Committee; posts must follow the Code of Conduct, be relevant to the OpenSSF mission, and not promote vendor or commercial products. 
+
+To receive space in an OpenSSF newsletter, projects should contact the Marketing Committee a minimum of two weeks before the newsletter publication date. Projects are responsible for tracking due dates and submitting materials.
+Technical content should be reviewed by the TAC, and potentially other WG's or Committees as appropriate. This includes, but is not limited to: white papers, technical briefings, policy recommendations, etc. This does not include project-specific announcements, such as those relating to project milestone releases. 
```

Comment:
Note: this is a large change from the existing governance, which explicitly allows WGs to publish white papers without TAC approval, although review is encouraged: https://github.com/ossf/tac/blob/main/working-group-abilities.md#papers

I'd prefer to keep it this way and avoid unnecessary approvals.

Reply:
Here are a few examples of content that groups have published that I think they should be able to continue publishing:

https://slsa.dev/blog/2022/06/slsa-ssdf
https://slsa.dev/blog/2022/05/slsa-sbom
https://openssf.org/blog/2022/04/19/your-favorite-software-repositories-now-working-together/
https://blog.sigstore.dev/privacy-in-sigstore-57cac15af0d0

Reply:
Is this line scoped to just what is included in an OpenSSF newsletter?

Reply:
If it is, and that implicitly includes blogs, then I would be opposed to inserting TAC or even WG approval in the critical path to publishing, as nearly any piece of content from us will have something that would likely be considered technical, even a milestone release announcement. I think the OpenSSF staff have done a competent job of getting expert review for content when submitted to us on an ad-hoc basis. THough I go back to suggesting that the process for submitting content for the blog or newsletter or other channel should be out of bounds.

Reply:
What possibly IS in bounds would be to say that any Technical Deliverable, such as a White Paper or Guide or other formal work, require Working Group approval before being published as final. Though I agree with Dan to wishing to avoid critical-pathing on the TAC for too much.

Reply:
I think notification would probably address the major pain point people are experiencing and make them want to have an approval process: where did that come from? I had no clue...

Reply:
I agree with @lehors: a notification is most likely what we are looking for here. Approval is too strict, even implying that the TAC member are supposed to in-depth review proposals coming in. However, as pointed out by @brianbehlendorf above, the "TAC is there to make sure the trains run on time and generally speaking to the right destination", so I think it is a valid request to actively point the TAC to upcoming publications. This could even be done as simply as posting an announcement on the (private - if we don't want to leak information about an upcoming publication) TAC mailing list.

Reply:
There have been frequent surprises to both TAC and Board members by publications coming from WGs, which raised serious questions. 

tldr; when content is presented through the megaphone of the OpenSSF's official publication channels, it gains an air of importance not commensurate to the level of responsibility any single Working Group has been granted. Neither is the current model generalizable: some WGs have excellent participation while others have inconsistent and imbalanced participation. We cannot, today, make a policy that all WGs are free to publish content without an approval process, and I hesitate to bless today's most active WGs while disadvantaging others in an unequal way.

The process does not need to be heavy-handed, but I believe it needs to be fair to all members and participants, and respectful of the reputation of the organization as a whole.

Reply:
> tldr; when content is presented through the megaphone of the OpenSSF's official publication channels,

The scope question Bob raised is my biggest one - what do we consider OpenSSF official channels? Is that just the openssf blog and newsletter today?

Reply:
We also have a Twitter account, a LinkedIn presence, I believe a FB group, a WeChat account, as well as of course the conferences under our name and any time someone asks us to suggest someone to speak at a conference on our behalf. 

I respect the TAC's wishes to be given a chance to review important technical content the WGs publish, and believe some process for the most important items and channels is merited. Some communities ask that things published as an "OpenSSF White Paper" be reviewed by their TOC/TAC/TSC etc; others ask projects to get approval before publishing a 1.0 release. There is an inherent tension between centralization and "being surprised" and I'd encourage TAC members to lean towards a willingness to be surprised occasionally (we can always issue corrections or add remediations) in the interests of agility/velocity.

Reply:
I agree and think the TAC should remain out of the critical path for publications.

Reply:
I am delighted when projects have their own successful brand and publish project-specific content through project-specific channels. 

The distinction between project brand and openssf brand should be clarified, and that is part of my intent here. To the examples listed above, I have no concerns about sigstore publishing sigstore content on sigstore's blog, or slsa publishing slsa content on slsa's blog. But I do have concerns about any project publishing its content on the openssf official channels without peer-review. I would also have concerns if a project were to publish content that misused the openssf brand on its own channels (though I hope that never becomes an issue).


Why am I bringing all this up?  Organizations like the OpenSSF create a space for collaborative management of IP -- and that includes brand assets, non-technical content, and technical publications. Ergo, is an important function of this (and similar) organization to provide peer-review of technical content that represents the Foundation publicly. Again, my goal here is not to restrict what Projects publish under their own brand / channel / etc, but to ensure that the OpenSSF brand is used responsibly, fairly, and transparently by all members of the foundation. 

So, I'm advocating for the documentation & creation of a transparent and consistent process for approval of brand-associated technical content. 

Whether the approving body is the TAC or whether we adopt some other structure (I have yet to see any counter-proposals and would be happy to review something more tangible than "I object"), is less important to me than the attributes of such a process: clear, consistent, and representative. If you don't want the TAC to be in the critical path, please propose another structure that provides peer-review by a technical body that represents the openssf membership.

Reply:
Ah, forgot to say, a notification process would meet my goal if it were, again, transparent and consistent and representative.

For example,
- notification sent to the TAC list at the formation of a SIG whose intended output is a White Paper, inviting broad participation in the SIG
- notification to the TAC list sent at a later date , once there is a working draft ready for comment
- one week before publication, notice sent again to the TAC list with the final draft

Reply:
Lets start with notification first in this V1, and then we as TAC can discuss together if we need any other heavyweight, formal process. The notification can come with a time window, like X days before it gets published. This gives people time to raise concerns. I think it has worked for last few releases, we notified TAC, some WG stacks and everyone supported.

Reply:
If this is clearly scoped to just the openssf blog and the newsletter and projects can still use their own channels I have zero concerns.

I think that's all technically in the scope of the GB and marketing committee though today so I'm not sure we can require this without their approval, but I have no objections to it in theory.